src/styles/pointerEventsNone/index.ts

```diff
@@ -0,0 +1,7 @@
+import PointerEventsNone from './types';
+
+const pointerEventsNoneStyles: PointerEventsNone = {
```

Comment:
just came across this PR

let's use the same name for the same constant across platforms. You used `pointerEventsNone` for native and `pointerEventsNoneStyles` for default.

Although both are exported as a default export, it's better to use the same name (only implementation should differ across platforms)

Reply:
@pasyukevich friendly bump to address this comment!